File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string binomial) {
    int answer = 0;
    for(int i = 0; i < binomial.size(); i++){
        if(binomial[i] == '+'){
            return stoi(binomial.substr(0, i-1))+stoi(binomial.substr(i+2, binomial.size()-i+1));
        }
        else if(binomial[i] == '-'){
            return stoi(binomial.substr(0, i-1))-stoi(binomial.substr(i+2, binomial.size()-i+1));
        }
        else if(binomial[i] == '*'){
            return stoi(binomial.substr(0, i-1))*stoi(binomial.substr(i+2, binomial.size()-i+1));
        }
    }
    return answer;
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    answer = 0
    n1,op,n2 = binomial.split()
    if op == '+':
        return int(n1) + int(n2)
    elif op == '-':
        return int(n1) - int(n2)
    elif op == '*':
        return int(n1) * int(n2)
    return answer